WinRT.Host/NetHost.targets

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ResolveNuGetPackages>false</ResolveNuGetPackages>
+    <BuildPassReferences>true</BuildPassReferences>
+    <NetHostUseAsStatic>true</NetHostUseAsStatic>

```

Comment:
Who is meant to set this?  Is this intended for us or for someone to override?

Reply:
this is an internal setting for winrt.host.dll, to eliminate yet another dll distribution (nethost.dll)